x-pack/legacy/plugins/uptime/server/graphql/monitor_states/resolvers.ts

```diff
@@ -39,9 +39,9 @@ export const createMonitorStatesResolvers: CreateUMGraphQLResolvers = (
   return {
     Query: {
       async getMonitorStates(
-        resolver,
+        _resolver,
```

Comment:
any reason why this now has an underscore in front?

Reply:
It's a typescript thing. You can resolve unused parameter warnings by prefixing them with `_`.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html search for "exempt from unused parameter checking" to see more info.